scripts/pages/zoning/zoning.ts

```diff
@@ -53,13 +53,29 @@ export enum RegionPolygonProblem {
 export enum ItemType {
 	TRACK = 0,
 	SEGMENT = 1,
-	ZONE = 2
+	ZONE = 2,
+
+	GLOBAL_REGION_TYPE = 3, // the ItemType is itself a type (of global region)
+	GLOBAL_REGION = 4
+}
+
+export enum GlobalRegionType {
+	TYPES_LIST = 0,
+	ALLOW_BHOP = 1,
+	CANCEL_TIMER = 2
+}
+
+interface GlobalRegionSelection {
+	type?: GlobalRegionType;
+	regions?: Region[];
+	index?: number;
 }
 
 interface ZoneSelection {
-	track: MainTrack | BonusTrack | null;
-	segment: Segment | null;
-	zone: Zone | null;
+	track?: MainTrack | BonusTrack | null;
+	segment?: Segment | null;
+	zone?: Zone | null;
+	global_region?: GlobalRegionSelection | null;
```

Comment:
Why does this have an underscore? Everything else is using camelCase.

Reply:
Fixed